src/selectors/session.js

```diff
@@ -16,6 +16,11 @@ const prompt = createSelector(
   (promptIndex, stage) =>  stage.params.prompts[promptIndex]
 )
 
+export const getStages = createSelector(
```

Comment:
can we use the same naming as the rest of the methods here: i.e `stages`, and refactor `stage` to use this method.